src/common/libraries/MessageLib.sol

```diff
@@ -10,8 +10,6 @@ import {AssetId} from "src/common/types/AssetId.sol";
 enum MessageType {
```

Comment:
Should we add a comment here, should never exceed 254 messages, to not re-use the message proof ID accidentally? Very unlikely but just to have documented.

Reply:
There is a test to avoid that, but I agree in adding it